djgeojson/serializers.py

```diff
@@ -363,14 +370,21 @@ def serialize(self, queryset, **options):
 
         self.start_serialization()
 
-        if isinstance(queryset, ValuesQuerySet):
-            self.serialize_values_queryset(queryset)
+        if django.VERSION >= (1, 9):
```

Comment:
If you do `ValuesQuerySet = QuerySet` for Django 1.9 you don't have to do this check and the same code will work for both.